src/Verify/VirtualizedRunHelper.cs

```diff
@@ -12,82 +14,77 @@ class VirtualizedRunHelper
     };
 
     public VirtualizedRunHelper(Assembly userAssembly)
+        : this(GetOriginalCodeBaseRoot(userAssembly),
+            Environment.CurrentDirectory,
+            Path.DirectorySeparatorChar,
+            path => File.Exists(path) || Directory.Exists(path))
+    {
+    }
+
+    private static string GetOriginalCodeBaseRoot(Assembly userAssembly) =>
+        AttributeReader.TryGetSolutionDirectory(userAssembly, false, out var solutionDir)
+            ? solutionDir : AttributeReader.GetProjectDirectory(userAssembly);
```

Comment:
Careful - the https://github.com/VerifyTests/Verify/pull/776 stack overflow fix is not contained here